iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/LoadingLimitsUtil.java

```diff
@@ -113,4 +119,30 @@ public static <L extends LoadingLimits, A extends LoadingLimitsAdder<L, A>> A in
                         .endTemporaryLimit());
         return adder;
     }
+
+    /**
+     * <p>Copies every limit from each operational limits group.</p>
+     *
+     * @param copiedBranch the copied object
+     * @param branch the object on which the copied object attributes are copied
+     */
+    public static <I extends Branch<I>> void copyBranchOperationalLimits(I copiedBranch, I branch) {
```

Comment:
Another thing I missed: could you rename this method in `copyOperationalLimits`?
There is no need to indicate the type of the parameters in the name.